### DISCUSSION TITLE
Please REVERT GitHub's New UI changes

### DISCUSSION THREAD

**Discussion by reeveng** [community/community#70249] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Dear [Thomas Dohmke](mailto:ashtom@github.com),

As Github's CEO, I hope this message finds you well. I am writing as a dedicated GitHub user and member of the developer community to express my concerns regarding the recent changes made to GitHub's navigation, code search, and homepage. While I understand that updates are often necessary, the recent changes have had a negative impact on the user experience, and I would like to respectfully request a reconsideration of these alterations.

The new navigation interface is less intuitive and has increased the complexity of tasks that were previously straightforward. The code search functionality, which is a crucial part of our daily development workflow, has become less efficient and less accurate. As a result, it has become more challenging for users to find and contribute to open-source projects effectively.

Moreover, the changes to the homepage have made it less informative and less useful for tracking important repositories and activities. These modifications have disrupted our workflow and slowed down our development processes.

I understand that updates are made with the intention of improving the platform, but in this case, it appears that the changes have had the opposite effect. Many developers within the GitHub community share these concerns, and we believe that a reversion to the previous interface would greatly benefit us.

I kindly request that GitHub considers revisiting the recent updates to these features, taking into account the feedback from the user community. The prior interface was efficient and user-friendly, and it served the development community well.

Thank you for your attention to this matter. GitHub has always been a platform that values the input of its users, and I hope that our feedback can contribute to a more positive and productive experience for all developers.

Sincerely,

reeveng

**Comment by kubo6472** [comment]
I love how formally written this seems. If you've used GitHub's own copilot to generate this, I'd applaud you.
